### ISSUE TITLE
ALARM opcode

### ISSUE THREAD

**Issue by stephantual** [ethereum/go-ethereum#117]
We dapp developers require an ALARM opcode to schedule contracts 'waking up' and executing specific functions at regular, predetermined intervals.

Furthermore, ALARM is the simplest way to implement truly independent agents on the ethereum blockchain. 

We realize one alternative approach to this is the incentivization of users to schedule things in a centralized way in exchange of cryptotokens, however, we also do believe that is far fetched in the same way bitcoins SPV clients on ethereum are IOHO far-fetched.

TL;DR: can we please get the ALARM opcode? Thank you :)

**Comment by jdmaresco** [comment]
@obscuren Was there a write up anywhere of why this was decided against? Was it a permanent decision or are there any plans to implement it later?

**Comment by fjl** [comment]
No decision has been made yet. It is very likely that ALARM will be included in a future version of Ethereum. The issue was closed because protocol changes are not discussed here.

**Comment by jdmaresco** [comment]
Thanks @fjl. Curious where is the official place to discuss? Forum? Another specific repo?

**Comment by Gustav-Simonsson** [comment]
@jdmaresco this forum is welcome of protocol proposals: https://forum.ethereum.org/categories/protocol-and-client-discussion

**Comment by jdmaresco** [comment]
Thanks @Gustav-Simonsson

**Comment by zpsmcdev** [comment]
Acceptance criteria completed by PR #118 (368beb9a9). Both beacon Compose files publish 13000/tcp and 13000/udp; the live M1/M2 joiner sync verification is recorded in #77. Closing this issue as completed.
